apps/whale-api/src/module.indexer/model/dftx/update.masternode.spec.ts

```diff
@@ -85,8 +85,8 @@ describe('Update masternode', () => {
       rewardAddress: addressDest.utf8String,
       creationHeight: 103,
       resignHeight: -1,
-      resignTx: expect.stringMatching(/[0-f]{64}/),
-      collateralTx: expect.stringMatching(/[0-f]{64}/),
+      resignTx: expect.stringMatching(/[0-9A-Fa-f]{64}/),
```

Comment:
```suggestion
      resignTx: expect.stringMatching(/[0-9a-f]{64}/),
```

I'm assuming it's lowercaps only?

Reply:
Yup that's correct. Fixed in d5c83a5f